### ISSUE TITLE
Verbose doesn't really show much verbosity

### ISSUE THREAD

**Issue by muschellij2** [CenterForOpenScience/osfr#104]
So downloading large files with `osf_download` would probably help with `httr::progress()` (though it adds a dependency) when `verbose = TRUE`.  This can be implemented as in # https://github.com/ropensci/crul#progress-bars, but it seems like you'd need to add a passthrough to `.wb_cli` from `.wb_request` or `cli$verb`:

``` r
library(osfr)
x = osf_retrieve_file("https://osf.io/btgx3/") 
x %>%
  osf_download(verbose = TRUE)
#> Downloaded OSF file to Study_19_Figure_1.pdf
#> # A tibble: 1 x 4
#>   name               id                   local_path          meta         
#> * <chr>              <chr>                <chr>               <list>       
#> 1 Study_19_Figure_1… 5751d71d9ad5a102079… Study_19_Figure_1.… <named list …
```

<sup>Created on 2019-08-19 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> ─ Session info ──────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 3.6.0 (2019-04-26)
#>  os       macOS Mojave 10.14.6        
#>  system   x86_64, darwin15.6.0        
#>  ui       X11                         
#>  language (EN)                        
#>  collate  en_US.UTF-8                 
#>  ctype    en_US.UTF-8                 
#>  tz       America/New_York            
#>  date     2019-08-19                  
#> 
#> ─ Packages ──────────────────────────────────────────────────────────────
#>  package     * version    date       lib
#>  assertthat    0.2.1      2019-03-21 [1]
#>  backports     1.1.4      2019-04-10 [1]
#>  callr         3.3.1      2019-07-18 [1]
#>  cli           1.1.0      2019-03-19 [1]
#>  crayon        1.3.4      2017-09-16 [1]
#>  crul          0.8.0      2019-06-28 [1]
#>  curl          4.0        2019-07-22 [1]
#>  desc          1.2.0      2019-07-10 [1]
#>  devtools      2.1.0      2019-07-06 [1]
#>  digest        0.6.20     2019-07-04 [1]
#>  dplyr         0.8.3      2019-07-04 [1]
#>  evaluate      0.14       2019-05-28 [1]
#>  fansi         0.4.0      2018-10-05 [1]
#>  fs            1.3.1      2019-05-06 [1]
#>  glue          1.3.1      2019-03-12 [1]
#>  highr         0.8        2019-03-20 [1]
#>  htmltools     0.3.6      2017-04-28 [1]
#>  httpcode      0.2.0      2016-11-14 [1]
#>  jsonlite      1.6        2018-12-07 [1]
#>  knitr         1.24       2019-08-08 [1]
#>  magrittr      1.5        2014-11-22 [1]
#>  memoise       1.1.0      2017-04-21 [1]
#>  osfr        * 0.2.4      2019-08-19 [1]
#>  pillar        1.4.2      2019-06-29 [1]
#>  pkgbuild      1.0.3      2019-03-20 [1]
#>  pkgconfig     2.0.2      2018-08-16 [1]
#>  pkgload       1.0.2      2018-10-29 [1]
#>  prettyunits   1.0.2      2015-07-13 [1]
#>  processx      3.4.1      2019-07-18 [1]
#>  ps            1.3.0      2018-12-21 [1]
#>  purrr         0.3.2      2019-03-15 [1]
#>  R6            2.4.0      2019-02-14 [1]
#>  Rcpp          1.0.2      2019-07-25 [1]
#>  remotes       2.1.0      2019-06-24 [1]
#>  rlang         0.4.0      2019-06-25 [1]
#>  rmarkdown     1.14       2019-07-12 [1]
#>  rprojroot     1.3-2      2018-01-03 [1]
#>  sessioninfo   1.1.1      2018-11-05 [1]
#>  stringi       1.4.3      2019-03-12 [1]
#>  stringr       1.4.0      2019-02-10 [1]
#>  testthat      2.1.1      2019-04-23 [1]
#>  tibble        2.1.3      2019-06-06 [1]
#>  tidyselect    0.2.5      2018-10-11 [1]
#>  triebeard     0.3.0      2016-08-04 [1]
#>  urltools      1.7.3      2019-04-14 [1]
#>  usethis       1.5.1.9000 2019-08-15 [1]
#>  utf8          1.1.4      2018-05-24 [1]
#>  vctrs         0.2.0      2019-07-05 [1]
#>  withr         2.1.2      2018-03-15 [1]
#>  xfun          0.8        2019-06-25 [1]
#>  yaml          2.2.0      2018-07-25 [1]
#>  zeallot       0.1.0      2018-01-28 [1]
#>  source                                    
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  Github (muschellij2/desc@b0c374f)         
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  Github (centerforopenscience/osfr@c0441f9)
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  local                                     
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#>  CRAN (R 3.6.0)                            
#> 
#> [1] /Library/Frameworks/R.framework/Versions/3.6/Resources/library
```

</details>

**Comment by aaronwolen** [comment]
Progress bars are already implemented in the [next version](https://github.com/CenterForOpenScience/osfr/tree/multi-file-downloads), along with some improvements to the messaging:

```r
osf_retrieve_file("https://osf.io/btgx3/") %>%
+   osf_download(verbose = TRUE)

## Downloaded file to ./Study_19_Figure_1.pdf

## # A tibble: 1 x 4
##  name             id                local_path         meta    
##  <chr>            <chr>             <chr>              <list>  
## 1 Study_19_Figure… 5751d71d9ad5a102… ./Study_19_Figure… <list […
```

Let me know if you're interested in beta testing when the time comes.

**Comment by aaronwolen** [comment]
This was addressed [v0.2.5](https://github.com/CenterForOpenScience/osfr/releases/tag/v0.2.5). Feel free to reopen if you have any other suggestions.
